app/controllers/ticket_comments_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class TicketCommentsController < ApplicationController
+  include Rails::Pagination
+
+  before_action :authenticate_user!
+
+  def index
+    ticket = Ticket.find(params.require(:ticket_id))
+
+    # Currently only stakeholders can access the ticket comments.
+    return head :unauthorized unless ticket.can_user_access?(current_user)
+
+    comments = ticket.ticket_comments.order(created_at: :desc)
+    paginate json: comments
```

Comment:
This pagination is kinda pointless if you don't query for individual pages in the frontend.

Reply:
Okay removed pagination, will implement this in frontend later and add it.